src/Chunk.ts

```diff
@@ -448,15 +447,14 @@ export default class Chunk {
 				: options.entryFileNames;
 			const currentDir = dirname(sanitizedId);
 			const fileName = renderNamePattern(
-				pattern,
+				typeof pattern === 'function' ? pattern(this.getChunkInfo()) : pattern,
```

Comment:
this change isn't needed anymore. Happy to revert, but it makes `renderNamePattern` less coupled to chunks

Reply:
I think it's ok, let's keep it